### DISCUSSION TITLE
Symbols for cmd, shift in command palette

### DISCUSSION THREAD

**Discussion by rhsev** [wezterm/wezterm#4314] [Q&A] [answered]
I'm trying to get to know Wezterm. Consider me a rookie ... 
In the Command Palette there are strange symbols for cmd, shift etc. How can I change this?
I use Sonoma, Firacode Nerd Font, Nord theme and Fish and have not configured anything else.
<img width="951" alt="Bildschirmfoto 2023-09-19 um 10 31 35" src="https://github.com/wez/wezterm/assets/111519454/2c08b2e2-766f-4e04-8681-46af96fb3f0d">
Is there any advice (discussion, documentation) on how I can customise the look of the command palette?

**Comment by wez** [comment, marked as answer]
The symbols depend on the `Symbols Nerd Font Mono` font. That is normally bundled as part of wezterm, but some distributions choose to split that apart.  A further complication is that Nerd Fonts made incompatible changes between version 2.0 and 3.0.

So this issue is either caused by a missing font or a mismatch in the version of the font.  I'd recommend updating to the latest nightly version of wezterm, and if you installed `Symbols Nerd Font Mono` through your distribution, either remove it or update it to the latest version.

**Reply by rhsev** [reply]
Great. Thank you, Wez.
The upgrade from wezterm 20230712 to wezterm 20230920 (nightly) has helped. 
I did not need to change the font. The Finder shows FiraCode-Light.ttf V6.002.
As you can see, I was also able to adjust the colours of the command palette in the meantime. 
<img width="937" alt="Bildschirmfoto 2023-09-22 um 00 57 14" src="https://github.com/wez/wezterm/assets/111519454/9d3ef1eb-79ec-4f19-b680-25d83bcc40da">
